connector-api/src/main/java/it/pagopa/selfcare/dashboard/connector/model/user/User.java

```diff
@@ -0,0 +1,16 @@
+package it.pagopa.selfcare.dashboard.connector.model.user;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
```

Comment:
@KevinSi96 use @Data instead of @Getter and @Setter so Lombok will create the toString() method too